mods/voidmulti.sc2mod/base.sc2data/gamedata/abildata.xml

```diff
@@ -912,9 +1042,12 @@
         </InfoArray>
     </CAbilMorph>
     <CAbilEffectTarget id="TimeWarp"/>
+    <CAbilTrain id="TrainQueen">
+        <InfoArray index="Train1" Effect=""/>
+    </CAbilTrain>
     <CAbilEffectTarget id="Transfusion">
         <Effect index="0" value="TransfusionImpactSet"/>
-        <CmdButtonArray index="Execute" Requirements="QueenOnCreep"/>
+        <CmdButtonArray index="Execute" Requirements="OnCreep"/>
```

Comment:
Pointless change that introduces a text bug. See the comment about the Queen's unit behavior